### ISSUE TITLE
Cannot see file list from my EC2 instance.

### ISSUE THREAD

**Issue by vaibhav6233m** [jacklam718/ftp#2]
Hi @jacklam718,
I really liked you application, it works perfectly on the local LAN. But I tried to access my EC2 instance from my machine using the instance IP, it connects successfully and gives a popup of selecting the the options "Register" and "Visitor". I tried both the ways but there is always an exception of "socket.timeout: timed out". I can also see the request log (Username and Password entered) on the terminal for ftp_server.py

I have also opened the inbound ports from the security group. I tried it with the port 21 but I was getting permission denied. So I am using a port 1025 port(also changed the port from ftp_client.py). Is that a problem?

See the screeshots

![screenshot from 2018-01-30 11-23-01](https://user-images.githubusercontent.com/29979122/35550356-ff4317ca-05af-11e8-8318-7ffb4f983e5e.png)

![screenshot from 2018-01-30 11-09-41](https://user-images.githubusercontent.com/29979122/35550121-0f9c3e86-05af-11e8-9733-0cc958df9455.png)
![screenshot from 2018-01-30 10-58-33](https://user-images.githubusercontent.com/29979122/35550123-0fe0a314-05af-11e8-96ae-90b8fe31d234.png)

**Comment by jacklam718** [comment]
@vaibhav6233m still getting permission after you change port to 1025?

**Comment by vaibhav6233m** [comment]
No, permission issue was solved for port no 1025. But I am not able to see the file list of the remote server. That was the actual problem.
